utils/pvacapi/app.py

```diff
@@ -3,17 +3,18 @@
 import connexion
 import os
 import sys
-import http.server
-import socketserver
-import socket
 import time
+import argparse
 
-from webbrowser import open_new_tab
 from flask_cors import CORS
 from utils.pvacapi.controllers.utils import initialize
 from utils.pvacapi.controllers.utils import getIpAddress
+
 #FIXME: sanitize sample name
 def main():
+    parser = argparse.ArgumentParser(description='parse pvacapi arguments')
```

Comment:
The description actually gets output as part of the usage information. So this should be something more descriptive about pVACapi itself.